zest/releaser/vcs.py

```diff
@@ -18,6 +18,12 @@
 ['"]             # String literal begins
 \d               # Some digit, start of version.
 """, re.VERBOSE)
+UPPERCASE_VERSION_PATTERN = re.compile(r"""
+^                # Start of line
+VERSION\s*=\s*   # 'version =  ' with possible whitespace
```

Comment:
The comment should also be uppercased.

 # 'VERSION =  ' with possible whitespace